tox.ini

```diff
@@ -9,17 +9,7 @@ deps = keyring
        django
        webtest
        nose
+       coverage>=3.6,<3.99
        unittest2
 setenv = PYTHONPATH=../google_appengine
-
-[testenv:py26]
-commands = nosetests --ignore-files=test_oauth2client_appengine\.py
```

Comment:
Why is the loss of this --ignore-files bit okay? It looks important, it smells important, and it's completely unmentioned in the commit log message?